src/Windows/Avalonia.Win32/WindowImpl.cs

```diff
@@ -164,7 +165,7 @@ public WindowImpl()
 
             Screen = new ScreenImpl();
             _storageProvider = new Win32StorageProvider(this);
-
+            _inputPane = Win32Platform.WindowsVersion >= PlatformConstants.Windows10 ? new WindowsInputPane(this) : null;
```

Comment:
there is some necessary using:
```csharp
using Avalonia.Metadata;
using Avalonia.Rendering;
using Avalonia.Win32.DirectX;
using Avalonia.Win32.WinRT;
using System.Diagnostics;
using static Avalonia.Controls.Platform.IWin32OptionsTopLevelImpl;
```
missing  _inputPane?.Dispose() in Dispose method.